src/ORM/DataList.php

```diff
@@ -203,10 +203,14 @@ public function sql(&$parameters = [])
     /**
      * Return a new DataList instance with a WHERE clause added to this list's query.
      *
+     * This method accepts raw SQL so is vulnerable to SQL injection attacks,
+     * it's preferable to use filter() instead which does not allow raw SQL
```

Comment:
Could we use something like this? The method is fine as long as the devs sanitise their inputs properly and don't pass in raw values from e.g. forms etc.

```suggestion
     * This method accepts raw SQL so could be vulnerable to SQL injection attacks if used incorrectly,
     * it's preferable to use filter() instead which does not allow raw SQL.
```

Reply:
I second this - obviously the same applies where similar messages are added to other phpdocs in this PR.

Reply:
Updated